docs/USER.md

```diff
@@ -156,9 +158,82 @@ type: Opaque
 stringData:
   tokenData: $AT_DATA
 ```
-After reconcillation SPIAccessToken should be filled with the Access Token metadata, it's `status.Phase` should be `Injected` and upload Secret is removed.
+After reconciliation SPIAccessToken should be filled with the Access Token metadata, it's `status.Phase` should be `Injected` and upload Secret is removed.
 In a case if something goes wrong the reason will be written to K8s Event, you can check it with `kubectl get event $upload-secret`.
 
+## Providing secrets to a service account
+
```

Comment:
Would not it be more clear for user to have two different fields (to put one of it), like:
- ServiceAccount: to define and create SA, always managed
- LinkedServiceAccount: with a reference to existed SA (+ linkSecretAs etc)
?

Reply:
This could mean that users might think that both can be used at the same time.

Reply:
@xbaran4 well, we need to explain this in doc, thets exactly my point, I think user have much more chances to be confused if we have it under the same field.